test/app/attachments_test.js

```diff
@@ -0,0 +1,105 @@
+const fse = require('fs-extra');
+const path = require('path');
+const stringToArrayBuffer = require('string-to-arraybuffer');
```

Comment:
Looks to me like we're pulling this in for [a very simple function](https://github.com/dfcreative/string-to-arraybuffer/blob/b52f48da7731a3515384d013143e9bfe6d4f8828/index.js#L23-L29):

```
function str2ab (str) {
	var array = new Uint8Array(str.length);
	for(var i = 0; i < str.length; i++) {
		array[i] = str.charCodeAt(i);
	}
	return array.buffer
}
```

I feel like the risk we take on in adding another dependency (even if a dev dependency) is not worthwhile.

Reply:
Done!